Service/ConfigurationService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Becklyn\MysqlDumpBundle\Service;
+
+
+class ConfigurationService
+{
+    /**
+     * The DumperBundle Configuration
+     *
+     * @var array
+     */
+    private $configuration;
+
+
+    /**
+     * ConfigurationService constructor.
+     *
+     * @param $configuration
+     */
+    public function __construct ($configuration)
```

Comment:
You should make that smarter. Use two parameters and do sanitization of them right here.